library/periodic_badges.js

```diff
@@ -0,0 +1,62 @@
+const PeriodicBadges = {
```

Comment:
Think you may want to freeze and make this an export.

Reply:
Oh, looks like you beat me to it already.

Reply:
The wonder of Travis caught that one, yeah.